tests/AuthService.IntegrationTests/DataAccess/Providers/AuthServiceRoleProviderTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿using AuthService.BusinessLogic.Entities;
+using AuthService.BusinessLogic.Extensions;
+using AuthService.BusinessLogic.Models.Enums;
+using FluentAssertions;
+
+namespace AuthService.IntegrationTests.DataAccess.Providers
+{
+    public class AuthServiceRoleProviderTests : BaseTestClass
+    {
+        public AuthServiceRoleProviderTests(GrpcAppFactory factory) : base(factory)
+        {
+        }
+
+        [Fact]
+        public async Task GetAll_Should_Return_Result()
+        {
+            //Arrange
+            var count = _seedRoles.Count;
+
+            //Act
+            var actualResult = await RoleProvider.GetAll(CancellationToken);
+
+            // Assert
+            actualResult.Should().Equal(_seedRoles, (c1, c2) => c1.Name == c2.Name);
+            actualResult.Should().HaveCount(count, "because we have this count of seed roles");
+        }
+
+        private static readonly List<Role> _seedRoles = new()
```

Comment:
maybe we should make foreach on AuthRole. Because if we add new role into the seed we should support this test too (test will fail).

Reply:
Also we have same protected list in the test class. Looks like duplicate.